juju/python-libjuju
# chore: deprecate juju.jasyncio

#### Description

*\<I created a new file called _jasyncio, passed all functions from jasyncio to there, but maintained the jasyncio file with a deprecation warning. Then, I changed jasyncio to _jasyncio or asyncio in all files that it appeared.\>*

*\<Fixes: Stopped using jasyncio and imported asynchronous functions directly from asyncio\>*


#### QA Steps

*\<Commands / tests / steps to run to verify that the change works: I do not know yet, this is my first contribution to an open source project, and I could use some guidance.\>*

```
tox -e py3 -- tests/unit/...
```

```
tox -e integration -- tests/integration/...
```

All CI tests need to pass.

*\<Please note that most likely an additional test will be required by the reviewers for any change that's not a one liner to land.\>*

#### Notes & Discussion

*\<This is my first time contributing and I am still learning how. Can you help me telling how can I test if the changes work, if I should do more smaller commits or only big commits, or any other guidance of how to work on this project?\>*
